clickhouse/client.h

```diff
@@ -47,8 +47,23 @@ struct ClientOptions {
         return *this; \
     }
 
+
+    /// List of hostnames with service ports
+    struct HostPort {
+        std::string host;
+        std::optional<unsigned int> port;
+
+        explicit HostPort(std::string host, std::optional<unsigned int> port = std::nullopt) : host(std::move(host)), port(std::move(port)) {
+        }
+    };
+    DECLARE_FIELD(hosts_ports, std::vector<HostPort>, SetHost,{});
     /// Hostname of the server.
-    DECLARE_FIELD(host, std::string, SetHost, std::string());
+    std::string host = std::string();
```

Comment:
You need to remove it so not to duplicate. If you don't remove it somehow, you need to declare this field with DECLARE_FIELD

Reply:
I can't remove `host` because it is a public field.
I returned declaration by `DECLARE_FIELD`

Reply:
Well, ok, leave it as is for now. But it would be great to write it in the pull request description that the host field is "duplicated"

Reply:
Why this host is duplicated? Now host in `host` field isn't stored in `hosts_ports`.

Reply:
OK